Show energy flow amperage and correct voltage tier in WAILA for machine blocks

wlhlm opened the pull request:
Previously, WAILA just showed voltage tier based on total energy flow. This was problematic for machines that accept multiple amps such as the Arc Furnace. Even though it can accept up to 3A of input, increasing the total input voltage by one tier, it obviously cannot accept that voltage from a single source. Should hopefully reduce confusion.

Should apply to singles, multis, and MuTEs.

Suggested by @chill-was-taken

![2023-05-05_21 44 33](https://user-images.githubusercontent.com/1928113/236622447-8a0b2ba3-5f8b-4e7c-8071-ffef3e386409.png)

eigenraven commented on src/main/java/gregtech/api/util/GT_Utility.java:
```diff
@@ -465,6 +465,10 @@ public static byte getTier(long l) {
     }
 
+    public static long getAmperageForTier(long voltage, byte tier) {
+        return (voltage / GT_Values.V[tier]) + 1;
```
Use ceilDiv here for an integer division rounding up (it's defined in the same class)

wlhlm commented:
Addressed review feedback and added a special case for steam single blocks to continue using to old energy display in WAILA.

Only thing that will be slightly weird is GT++ steam multis as the GT Multi base class which those inherit from and which implements the WAILA handling does not natively support steam, so the Steam Multi Macerator for example will show amps in WAILA.

chill-was-taken commented:
Lightning-fast. 
It took less than three hours from the inquiry to the approved PR. 
@wlhlm, @eigenraven, you guys are awesome!

wlhlm commented:
Apologies there was a spurious line I forgot to remove. See [here](https://github.com/GTNewHorizons/GT5-Unofficial/compare/9f2971c2d12f9010f85a9c855019c6561cc18480..1fbbcba9c51e47b8649514e2cee77fbb4083b577). Please re-review.
